lib/elixir/lib/registry.ex

```diff
@@ -354,13 +354,20 @@ defmodule Registry do
             "expected :listeners to be a list of named processes, got: #{inspect(listeners)}"
     end
 
+    compressed = Keyword.get(options, :compressed, false)
```

Comment:
This is a minor deviation from the `:ets` API which just does `[:compressed]` but that isn't conventional Elixir.